### PR TITLE
TELCODOCS-442, TELCODOCS-443: PTP 4.11 release notes

### PR DESCRIPTION
**Pull request by aireilly**
Adds release notes for the following PTP 4.11 doc epics: 
* https://issues.redhat.com/browse/TELCODOCS-442
* https://issues.redhat.com/browse/TELCODOCS-443

Version(s):
* enterprise-4.11

Preview:
* http://file.emea.redhat.com/aireilly/ptp-411-release_notes/release_notes/ocp-4-11-release-notes.html#ocp-4-11-technology-preview
* http://file.emea.redhat.com/aireilly/ptp-411-release_notes/release_notes/ocp-4-11-release-notes.html#ocp-4-11-networking-linuxptp
* http://file.emea.redhat.com/aireilly/ptp-411-release_notes/release_notes/ocp-4-11-release-notes.html#ptp-events-enhancements

### CODE REVIEW

**Review by sheriff-rh — APPROVED** [review]
LGTM!

**Review by aneeshkp — APPROVED** [review]

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -421,6 +421,20 @@ With this update, the External DNS Operator provides the following enhancements:
...
+You can now configure dual network interfaces (NICs) as boundary clocks with `PtpConfig` profiles for each NIC channel.
+
+For more information, see xref:../networking/using-ptp.adoc#ptp-dual-nics_using-ptp[Using PTP with dual NIC hardware].
+
+[id="ocp-4-11-networking-linuxptp-events"]
+==== PTP events enhancements
+
+A new PTP events API endpoint is available, `api/cloudNotifications/v1/publishers`. Using this endpoint, you can get PTP `os-clock-sync-state`, `ptp-clock-class-change`, and `lock-state` details for the cluster node.
+
+For more information, see xref:../networking/using-ptp.html#cnf-fast-event-notifications-api-refererence_using-ptp[Subscribing DU applications to PTP events REST API reference].
```
Comment (bergerhoffer) [inline release_notes/ocp-4-11-release-notes.adoc:436]:
```suggestion
For more information, see xref:../networking/using-ptp.adoc#cnf-fast-event-notifications-api-refererence_using-ptp[Subscribing DU applications to PTP events REST API reference].
```

**Review by bergerhoffer — COMMENTED** [review]
LGTM now too, thanks! Merging

**Pull request merged**